ddtrace/internal/ci_visibility/utils.py

```diff
@@ -10,12 +10,25 @@
 log = get_logger(__name__)
 
 
-def get_source_file_path_for_test_method(test_method_object) -> typing.Union[str, None]:
+def get_relative_or_absolute_path_for_path(path: str, start_directory: str):
     try:
-        source_file_path = os.path.relpath(inspect.getfile(test_method_object))
+        relative_path = os.path.relpath(path, start=start_directory)
```

Comment:
feel free to ignore: would it make sense to use `pathlib` for these path operations at this point, or would it make more sense to do a refactor later?

Reply:
Seems like, with `pathlib`, we'd still end up with a `ValueError` for relative paths that can't be computed, so the logic would be the same.

For me, that adds up to "refactor later".